lightly/active_learning/agents/agent.py

```diff
@@ -119,29 +120,29 @@ def query(self, sampler_config: SamplerConfig, al_scorer: Scorer = None) -> List
                            "including the current labeled set "
                            "(sampler_config.n_samples) "
                             "is smaller than the number of samples in the current labeled set.")
```

Comment:
We should add to this warning that the sampler will skip and just return the already labeled set.